### ISSUE TITLE
error on vagrant provision

### ISSUE THREAD

**Issue by plandolt** [vagrant-php/ubuntu#19]
Steps to reproduce:
- do a fresh install of vagrant-php
- run vagrant up
- run vagrant provision

```
==> default: /home/vagrant/.bashrc: line 18: /home/vagrant/.application_aliases/*: No such file or directory
==> default: /root/.bashrc: line 18: /root/.application_aliases/*: No such file or directory
```

**Comment by dominikzogg** [comment]
@scuben wow github autoclosed this issue cause i called the commit fix #19 ;-)

**Comment by plandolt** [comment]
@dominikzogg intuitive! :)
